titration/ui_state/user_value/user_value.py

```diff
@@ -52,9 +52,10 @@ def handle_key(self, key):
         Parameters:
             key (char): the keypad input to determine which state to go to
         """
-        if key == "A":
+        if (key == "A") & (self.value not in ("", ".")):
```

Comment:
What does the "&" do here? I thought python used "and".

Reply:
Yes, I fixed this and implemented it with "and" instead.